status-im/help.status.im
# CI changes, update to Insiders build, and multiple minor changes

- Add Material insiders plugin: git-authors to mkdocs.insiders.yml
- Update requirements.txt to support Material Contributors extension
- 131 change your status display name (#152)
- ci: provide GH_TOKEN to install from private repo
- Draft landing page to test Material insiders' build
- test/provisional to check Material insiders build installation
- Clean up unused files and directories in docs repo
- Additing the mkdocs.yml file, deleting the part of the path (#163)
- Add placeholder doc and text to test Material admonitions and annotations
- 130 about status display names (#167)
- Update sections ToC and sections names to match task analysis mind-map
- 130 about status display names (#170)
- set up provisional landing page cards, texts, and links
- Remove comments (#) from urls-en.txt file - (#)s break file
- 169 your chatkey emojihash and identicon ring (#174)
- Add 'includes' directive to main index.md file for URL resolution
- Adding the anchor feature (#176)
- Adding the md_in_html feature into style guide config file (#177)
- Adding the inherit line into the style guide config file (#178)
- fix domain and path in site_url in configs
- Adding the permalink into the style-guide config file (#181)
- drop unnecessary /index from URLs
- remove doc reference and content for 'Join a Status Community'
- Add card grids to main page and update URLs file
- Add card grids to main page and update URLs file FIX
- Add Material for MkDocs requirements and additional info
- Change CONTRIBUTING.md title
- use requirements.txt in build docs
- rename status-react to status-mobile
- ci: use Dockerfile for Jenkins CI builds
